kura/org.eclipse.kura.core.net/src/main/java/org/eclipse/kura/core/net/WifiConfigurationInterpreter.java

```diff
@@ -175,7 +175,7 @@ private static WifiBgscan getBgScan(Map<String, Object> properties, String prefi
     }
 
     private static WifiCiphers getPairwiseCiphers(Map<String, Object> properties, String prefix) {
-        Object pairwireCiphersObject = properties.get(prefix + WIFI_PAIRWISE_CIPHERS_KEY);
+        Object pairwiseCiphersObject = properties.get(prefix + WIFI_PAIRWISE_CIPHERS_KEY);
         if (pairwireCiphersObject instanceof String) {
```

Comment:
The change is not complete, as there were obviously other references to the variable.

Reply:
It is a local variable used only in this method.